xbmc/threads/Thread.h

```diff
@@ -51,7 +51,7 @@ class CThread
   bool IsRunning() const;
 
   bool IsCurrentThread() const;
-  bool Join(unsigned int milliseconds);
+  bool Join(std::chrono::milliseconds duration);
```

Comment:
Why do you enforce `std::chrono::milliseconds` here instead of doing it similar to `Sleep()`?

Reply:
I figured it was less useful and the required changes (moving the method to the header) seemed unnecessary. I can update it though for uniformity if desired.